.azure-pipelines/Linux-CI.yml

```diff
@@ -85,7 +85,6 @@ jobs:
       python onnx/gen_proto.py -l
       python onnx/gen_proto.py -l --ml
       python onnx/backend/test/stat_coverage.py
-      backend-test-tools generate-data
```

Comment:
This did not generate anything... remove it because it is not worked. Instead, we only check uploaded/generated in release CIs